charts/argo-cd/values.yaml

```diff
@@ -2638,9 +2638,6 @@ repoServer:
 
 ## ApplicationSet controller
 applicationSet:
-  # -- Enable ApplicationSet controller
-  enabled: true
```

Comment:
Is it better to write changelog in README...? 🤔

Reply:
I would mention that even in readme as enteprise users with proxy repositories might be suprised that this is now always installed.

Reply:
@pdrastil 
Thank you for your comment. I will add 🙋

Reply:
I added in this commit https://github.com/argoproj/argo-helm/pull/2688/commits/822ab42195754e79af05f4adb783295c438e2f61 . 🫡